features/upgrade/sdn/policy-upgrade.feature

```diff
@@ -477,3 +477,173 @@ Feature: SDN compoment upgrade testing
+    And evaluation of `"policy-upgrade7"` is stored in the :proj2 clipboard
+    When I run the :label admin command with:
+      | resource | namespace       |
+      | name     | <%= cb.proj2 %> |
+      | key_val  | team=operations |
+    Then the step should succeed
+    Given I use the "<%= cb.proj1 %>" project
+    Given I obtain test data file "networking/list_for_pods.json"
+    When I run oc create over "list_for_pods.json" replacing paths:
+      | ["items"][0]["spec"]["replicas"] | 1 |
+    Then the step should succeed
+    Given a pod becomes ready with labels:
+      | name=test-pods |
+    And evaluation of `pod(1).ip_url` is stored in the :p1pod1ip clipboard
+    Given I obtain test data file "networking/networkpolicy/allow-ns-and-pod.yaml"
+    When I run the :create admin command with:
+      | f | allow-ns-and-pod.yaml |
+      | n | <%= cb.proj1 %>       |
+    Then the step should succeed
+    Given I use the "<%= cb.proj2 %>" project
+    Given I obtain test data file "networking/list_for_pods.json"
+    When I run oc create over "list_for_pods.json" replacing paths:
+      | ["items"][0]["spec"]["replicas"] | 1 |
+    Then the step should succeed
+    Given a pod becomes ready with labels:
+      | name=test-pods |
+    And evaluation of `pod(2).name` is stored in the :p2pod1name clipboard
+    Given I obtain test data file "rc/idle-rc-1.yaml"
+    When I run oc create over "idle-rc-1.yaml" replacing paths:
+      | ["items"][0]["spec"]["replicas"] | 1 |
+    Then the step should succeed
+    Given a pod becomes ready with labels:
+      | name=hello-idle |
+    And evaluation of `pod(3).name` is stored in the :p2pod2name clipboard
+    And I wait up to 20 seconds for the steps to pass:
+    """
+    When I execute on the "<%= cb.p2pod1name %>" pod:
+      | curl | -I | <%= cb.p1pod1ip %>:8080 | --connect-timeout | 5 |
+    Then the output should contain "200 OK"
+    Then the step should succeed
+    Given I use the "<%= cb.proj0 %>" project
+    When I execute on the "<%= cb.rsyslog_server_pod %>" pod:
+      | grep | -w | verdict=allow | /var/log/messages |
+    Then the output should contain "verdict=allow"
+    
+    Given I use the "<%= cb.proj2 %>" project
+    When I execute on the "<%= cb.p2pod2name %>" pod:
+      | curl | -I | <%= cb.p1pod1ip %>:8080 | --connect-timeout | 5 |
+    Then the output should not contain "200 OK"
+    Then the step should fail
+    Given I use the "<%= cb.proj0 %>" project
+    When I execute on the "<%= cb.rsyslog_server_pod %>" pod:
+      | grep | -w | verdict=drop | /var/log/messages |
+    Then the output should contain "verdict=drop"
+    """
+
+  # @author asood@redhat.com
+  # @case_id OCP-44978
+  @admin
+  @upgrade-check
+  @4.10 @4.9
+  @aws-ipi @gcp-ipi @vsphere-ipi @azure-ipi @baremetal-ipi @openstack-ipi
+  @aws-upi @gcp-upi @openstack-upi
+  @destructive
+  @network-ovnkubernetes
+  Scenario: Check network policy ACL logging works post upgrade -prepare
```

Comment:
``` - prepare ``` not needed here

Reply:
@anuragthehatter easy for eyes to distinguish between two phases.

Reply:
@asood-rh Your both phases are ```- prepare``` :)

Reply:
@anuragthehatter Fixed.